src/v2/ion/i18nTransformer.js

```diff
@@ -143,11 +143,41 @@ const I18N_OVERRIDE_MAPPINGS = {
   'reenroll-authenticator-warning.okta_password.credentials.passcode': 'oie.password.newPasswordLabel',
   'incorrectPassword': 'oie.password.incorrect.message',
 
+  // Base Attributes
```

Comment:
add a unit test in `i18nTransformer_spec.js`

Reply:
@pradeepdewda-okta What kind of test should I add? Some of the base attributes
e.g.
`'enroll-profile.userProfile.lastName': 'oie.user.profile.lastname',`
`'enroll-profile.userProfile.firstName': 'oie.user.profile.firstname',`
`'enroll-profile.userProfile.email': 'oie.user.profile.primary.email',`

 are in `i18nTransformer.js` already but aren't found in `i18nTransformer_spec.js`

Reply:
nit: Due to the nature of how many attributes we are transforming, consider moving these to a separate file that can be individually tested. That will give your team a better sense of assurance none of these attributes get remapped in the future, causing i18n or messaging regressions.

Reply:
I tried refactoring a bit in the last push:
- moved these mappings into their own file and export that object
- in `i18nTransformer.js`, imported said object
  - and in `geti18nKey`, checked if the object contains the key and sets the key if so